src/providers/ContentShareProvider/index.tsx

```diff
@@ -117,18 +117,28 @@ const ContentShareProvider: React.FC = ({ children }) => {
     return () => window.removeEventListener('unhandledrejection', cb);
   }, [isLocalShareLoading]);
 
-  const toggleContentShare = useCallback(async (): Promise<void> => {
-    if (!audioVideo) {
-      return;
-    }
+  const toggleContentShare = useCallback(
+    async (mediaUrl?: string): Promise<void> => {
```

Comment:
I think we need to make this a separate hook (`useStartFileShare` or something), as it is a breaking change otherwise

`onClick={toggleContentShare} `

will pass the event object to toggleContentShare. So this will break for anyone already using this syntax

Reply:
Added `toggleVideoFileShare(url: string)` to `useContentShareControls`. Personally, I think that toggle video share should still be one of `useContentShareControls`. But I'm open for ideas here.